src/skills-builder/skills-builder-modal/view-results/CarouselStack.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CardCarousel } from '@edx/paragon';
+import RecommendationCard from './RecommendationCard';
+
+const CarouselStack = ({ selectedRecommendations }) => {
+  const { name: jobName, recommendations } = selectedRecommendations;
+  const lobNames = Object.keys(recommendations);
+
+  const normalizeLOBName = (lobString) => {
+    const newLobString = lobString.split('_').map(word => word[0].toUpperCase() + word.slice(1).toLowerCase());
+    return newLobString.join(' ');
+  };
+
+  return (
+    lobNames.map(lob => (
+      <CardCarousel
+        key={lob}
+        ariaLabel="card carousel"
+        title={<h3>{normalizeLOBName(lob)} recommendations for {recommendations[lob].id} {jobName}</h3>}
```

Comment:
I think we need to rethink this. I'm a little worried that the way this is written would not support localization/i18n. Is there a way to create a templated string with placeholders?

Reply:
Yes, this is another good catch. We can at least translate the "recommendations for" text. The `{recommendation[lob].id` part just shouldn't be there and is a mistake.